### ISSUE TITLE
Category Image

### ISSUE THREAD

**Issue by aerograf** [mambax7/extgallery#4]
It would be nice to make an image selection when creating category (page category&albums)

**Comment by mambax7** [comment]
The image is being selected from the pictures of the album. 
But definitely something to consider in the future.

**Comment by aerograf** [comment]
Yes I know. But this function appears during the editing of the category / album. Why not make it right away?

**Comment by mambax7** [comment]
You can leave it as open, so this will be a reminder...

**Comment by aerograf** [comment]
Closed because I figured out how it works :-) Initially there are no images in the album. You just need to add a description to the category creation.
